contracts/Tokens/VTokens/VToken.sol

```diff
@@ -464,6 +516,11 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         totalBorrows = totalBorrowsNew;
         totalReserves = totalReservesNew;
 
+        if (currentBlockNumber - reduceReservesBlockNumber >= reduceReservesBlockDelta) {
+            reduceReservesBlockNumber = currentBlockNumber;
+            _reduceReservesFresh(totalReservesNew);
```

Comment:
A reminder: to avoid unexpected behaviours, `VBep20Delegator::_setImplementation`, `setReduceReservesBlockDelta`, `setAccessControlManager`, and `setProtocolShareReserve` should be executed in a single tx during the upgrade.

Reply:
https://github.com/VenusProtocol/vips/pull/67